packages/@lwc/synthetic-shadow/src/polyfills/event-target/polyfill.ts

```diff
@@ -30,10 +39,7 @@ function addEventListener(
     listener: EventListenerOrEventListenerObject,
     optionsOrCapture?: boolean | AddEventListenerOptions
 ) {
-    // TODO [#2134]: Delegate this validation to the browser instead of silently swallowing
-    if (!isValidEventListener(listener)) {
-        return;
-    }
+    assertValidEventListenerArgs(arguments);
```

Comment:
Rather than making assertions against the arguments, I think we should just pass them through all the way down to the native method via `method.apply(this, arguments)`.

Reply:
Let's also make sure to add `fix #2134` to the PR description to automatically close the issue once the PR is merged.